setup.py

```diff
@@ -51,13 +53,8 @@
     packages=find_packages(include=["trino", "trino.*"]),
     package_data={"": ["LICENSE", "README.md"]},
     description=about["__description__"],
-    long_description=textwrap.dedent(
-        """
-    Client for Trino (https://trino.io), a distributed SQL engine for
-    interactive and batch big data processing. Provides a low-level client and
-    a DBAPI 2.0 implementation.
```

Comment:
@aksakalli Can you change the leading statement of existing `README.md` to include this information.

So something like:

```
Client for Trino (https://trino.io), a distributed SQL engine for interactive and batch big data processing.
Provides a low-level client and a DBAPI 2.0 implementation and a SQLAlchemy adapter.
It supports Python>=3.7 and pypy.
```

Reply:
done in c521f75 let me know if it's ok, I can squash the commits

Reply:
Looks good, please squash and I'll merge.

Reply:
squashed as 78fb9b0